octopus.go

```diff
@@ -18,9 +18,15 @@ not, you can obtain one from Tidepool Project at tidepool.org.
 package main
 
 import (
+	"crypto/tls"
```

Comment:
I am using this [sublime add](https://github.com/DisposaBoy/GoSublime) in that runs a go fmt on save . With my current setup that *seems* to push the putting standard libraries to the top and others listed below